overrides.nix

```diff
@@ -45,6 +45,12 @@ in
       buildInputs = old.buildInputs ++ [ pkgs.libffi ];
     }
   );
+  
+  black = super.black.overrideAttrs (
+    old: {
+      buildInputs = old.buildInputs ++ [ pkgs.setuptools_scm ];
```

Comment:
`pkgs.setuptools_scm` -> `self.setuptools_scm` (or you can use the `addSetupTools` function: `black = addSetupTools super.black;`).

Reply:
Thanks!